tests/myapp.yml.dist

```
@@ -1,7 +1,7 @@
 parameters:
-  client_id: Your Commercetools client ID
-  client_secret: Your Commercetools client secret
-  project: Your Commercetools project key
+  client_id: Your Composable Commerce client ID
+  client_secret: Your Composable Commerce client secret
+  project: Your Project key
```

```suggestion
  project: Your Composable Commerce Project key
```
In this case I would prefer to spell it out, since we do so for the client ID + secret, and I think it's less confusing this way